include/igl/boundary_loop.cpp

```diff
@@ -131,7 +131,7 @@ IGL_INLINE void igl::boundary_loop(
 template <typename DerivedF, typename DerivedL>
 IGL_INLINE void igl::boundary_loop(
   const Eigen::MatrixBase<DerivedF>& F,
-  Eigen::PlainObjectBase<DerivedL>& L)
+  Eigen::MatrixBase<DerivedL>& L)
```

Comment:
Hmm  sorry, `MatrixBase` should be used for *inputs*, and `PlainObjectBase` should be used for *output* (see [FAQ](https://libigl.github.io/faq/)).

Reply:
Yup, just noticed this.. fixing